Tests/StreamChatTests/MockEnpointResponses/CurrentUser.json

```diff
@@ -66,4 +66,75 @@
           "created_at" : "2020-08-07T03:36:37.271234Z"
       },
   ],
+  "channel_mutes": [
```

Comment:
Actually I am missing payload for the whole `NotificationChannelMutesUpdatedEvent` event. I shared to you in internal discussion patch which introduces this JSON + Adds some sample test to it.